tsconfig.base.json

```diff
@@ -0,0 +1,25 @@
+// the base tsconfig file
```

Comment:
I guess, those two TypeScript config related file shouldn't be part of the final version of PR, right?

Reply:
Good catch. I've removed them.